supp2u/src/components/Business/BusinessHeader.js

```diff
@@ -23,7 +23,7 @@ function BusinessHeader(props) {
           {props.info[0].street}, {props.info[0].city}, {props.info[0].zipcode}, {props.info[0].state}
           </div>
           <div className="ratings">
-          Rating: {props.info[1].data}
+          Rating: {props.info[0].data}
```

Comment:
Was there a reason this data was changed I did not see a change on the BusniessSingleView that is passing the props down to reflect this change.

Reply:
me either but I'm about to test it to see when I merge, if broken, i will change back

Reply:
me either, but I'm about to test it